services/has-many-getter.js

```diff
@@ -99,7 +85,6 @@ function HasManyGetter(model, association, opts, params) {
       .then(function (record) {
         return record['get' + _.capitalize(params.associationName)]({
           scope: false,
-          attributes: getSelect(),
```

Comment:
This is to remove the issue with Smart Fields computed with attributes that have their columns hidden.